.github/workflows/job-dormant-user-accuracy-report.yml

```diff
@@ -26,9 +26,10 @@ jobs:
       - name: Execute dormant user removal accuracy report run
         run: pipenv run python3 -m bin.dormant_user_removal_accuracy_report
         env:
-          ADMIN_GITHUB_TOKEN: ${{ secrets.GH_BOT_AUDIT_LOG_PAT_TOKEN }}
+          GH_MOJ_ADMIN_TOKEN: ${{ secrets.READONLY_MOJ_GITHUB_DORMANT_USERS }}
+          GH_AP_ADMIN_TOKEN: ${{ secrets.READONLY_AP_GITHUB_DORMANT_USERS }}
           ADMIN_SLACK_TOKEN: ${{ secrets.ADMIN_SEND_TO_SLACK }}
-          SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
+          GH_GITHUB_TOKEN_AUDIT_LOG_TOKEN: ${{ secrets.GH_BOT_AUDIT_LOG_PAT_TOKEN }}
```

Comment:
Is this the token that has to be classic to have the enterprise permissions?

Reply:
Yes I believe so, copied from the dormant user workflow